ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -172,6 +175,7 @@ public void configure(final KsqlConfig config) {
     }
 
     ksqlConfig = config;
+    ksqlRestConfig = new KsqlRestConfig(ksqlConfig.originals());
```

Comment:
Nit: You can just pass this in the constructor when created in `KsqlRestApplication` rather than creating it here.